roles/github/defaults/main.yml

```diff
@@ -184,6 +185,13 @@ github_run_overcloud_provision:
   arguments: "{{ github_kayobe_task_arguments }}"
   concurrency_group: overcloud
 
+github_run_pulp_container_sync:
+  file_name: run-pulp-container-sync.yml
```

Comment:
Would it be helpful to provide a single workflow that supports syncing, publishing, promoting packages & containers, with checkboxes for each?